README.md

```diff
@@ -26,14 +26,14 @@ You can view detailed documentation and guides at [https://ray-project.github.io
 #### Nightly version
 
 ```
-kubectl apply -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources"
+kubectl create -k "github.com/ray-project/kuberay/manifests/cluster-scope-resources"
```

Comment:
em. We can use create at this moment. The downside of `create` here is cluster-scope-resources has `ray-system` namespace as well. If user has the namespace in the cluster, upgrade will fail due to existence.

Let me check if we can resolve the issue by limiting the crd size later. This looks good to me at this moment